pkg/kubelet/volumemanager/cache/actual_state_of_world.go

```diff
@@ -268,6 +268,13 @@ type mountedPod struct {
 	// mapper used to block volumes support
 	blockVolumeMapper volume.BlockVolumeMapper
 
+	// spec is the volume spec containing the specification for this volume.
+	// Used to generate the volume plugin object, and passed to plugin methods.
+	// In particular, the Unmount method uses spec.Name() as the volumeSpecName
+	// in the mount path:
+	// /var/lib/kubelet/pods/{podUID}/volumes/{escapeQualifiedPluginName}/{volumeSpecName}/
+	spec *volume.Spec
```

Comment:
Could we name this volumeSpec? Not to be confused with pod spec

Reply:
I agree. To improve readability we should name this `volumeSpec`. Currently the code reads - `mountedPod.Spec` which almost sounds like it is pod spec.

Reply:
I followed the name from desired state data structure. If preferred, I can change in both places?

Reply:
I am happy to have just this one renamed to `volumeSpec` but your call.